spec/grape-apiary/sample_generator_spec.rb

```diff
@@ -36,4 +36,24 @@
       expect(JSON.parse(subject.response)['id']).to_not be(nil)
     end
   end
+
+  context '#sample' do
+    context 'when the JSON API entity exists' do
+      let(:entity) { V20170505::Entities::Widget }
+
+      it 'is the entity hash' do
```

Comment:
I'm not happy with this but my null object solution failed and I'm not going to reimplement the API and the grape jsonapi entities gem in this gem to test this method.